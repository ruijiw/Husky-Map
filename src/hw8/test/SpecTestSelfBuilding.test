# Test the program can find the correct path from a certain building to itself
r
MUS
MUS

# complete this test
q