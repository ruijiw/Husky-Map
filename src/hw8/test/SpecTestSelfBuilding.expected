Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): # Test the program can find the correct path from a certain building to itself
Abbreviated name of starting building: Abbreviated name of ending building: Path from Music Building (Northwest Entrance) to Music Building (Northwest Entrance): 
Total distance: 0 feet

Enter an option ('m' to see the menu): 
# complete this test